stable/app/values.yaml

```diff
@@ -35,11 +35,11 @@ migration:
 
 service:
   type: ClusterIP
-  port:
-    port: 80
```

Comment:
This port value is also used in `ingress.yaml`. We need to make changes there as well.

Reply:
Have made the necessary changes.